custom/icds/messaging/indicators.py

```diff
@@ -201,7 +201,7 @@ def get_value_from_fixture(self, fixture, attribute):
                         "Attribute {} not found in restore for AWC {}".format(attribute, location_name)
                     )
 
-        raise IndicatorError("AWC {} not found in the restore".format(location_name))
+        return 0
```

Comment:
from what i can tell, for all four of the ucrs being used, when the AWC id is not found it means that there were no forms or cases for that AWC that match the ucr filters, so the aggregation for these fields should be represented with a 0 instead of raising an exception